### ISSUE TITLE
Dont found anything and dont stop pokebuddy

### ISSUE THREAD

**Issue by danielcalcina** [finndev/PokeBuddy#779]
Finds nothing and even clicking STOP it keeps running.
![pkn](https://cloud.githubusercontent.com/assets/17730537/17568048/608057d6-5f18-11e6-9a12-af233bbe573f.png)

**Comment by munchieiam** [comment]
what worked for me was closing out of the app, rebooting, then opening it back up, going to settings, change the starting spot a little and then hitting start again. been working fine for a while now.

**Comment by 404unknown** [comment]
Please read previous issues before opening your own. There are already tons of issues talking about this. Your probably Ip-banned. If "stop" button doesnt work just close the exe

**Comment by Clax9** [comment]
@munchieiam Tried that as well, didn't work. Can confirm it's not a soft ban either. I can log into the game and catch pokemon just fine. Got it to work for a little bit then it just stopped.

**Comment by danielcalcina** [comment]
It is not the IP, the @munchieiam solution worked perfectly, Thanks

**Comment by danielcalcina** [comment]
You need to change the starting position

**Comment by reykime** [comment]
@munchieiam solution worked perfectly

**Comment by 5P4C3M4N** [comment]
Same issue, and same repeating 17 meter navigation prompt. Tried three different starting points and the issue persists.

**Comment by Dac93** [comment]
Same issue, I dont know what to do.

**Comment by munchieiam** [comment]
have you changed your walk speed at all? i usually run mine at 5-8 for a little bit to make it at least look more natural.

**Comment by Clax9** [comment]
@munchieiam Yes, just tried with a fresh download, new spot change not too far away and with different low speed. No pokestops. Still no soft ban, if it's an IP ban I tried restarting my modem and router too. I don't want to try using a VPN if I'm just going to get stopped in less than 5 minutes again.

**Comment by 5P4C3M4N** [comment]
Tried the same, reinstall/walkspeed adjust/starting location change/ settings in general. I have noticed it does not seem to say (Farming Pokestops) in the Current Tasks indicator even whilst it is active. Seems to have happened as soon as the update incorporating Incubator use occurred.

**Comment by Skolzack** [comment]
@5P4C3M4N I'm using the version of yesterday that incorporated the Incubator function and works fine. But I didn't closed my Pokebuddy since the Incubator update that was more than 24hour ago. It's strange and the dev's say nothing about this, only updates with no update notes.

**Comment by munchieiam** [comment]
i did see a previous post to un-check the incubate option. I am running the 0.0.8.4 version.

**Comment by 5P4C3M4N** [comment]
AH! Got it! Try going to settings, and have either use incubators selected or only have use permanent incubator selected. Basically only have one incubator option selected at a time. Having neither selected, or both selected at the same time seems to cause an issue.

**Comment by munchieiam** [comment]
nice find.

**Comment by 5P4C3M4N** [comment]
Hoping that resolves the issue for others as well, I'll keep testing to see if I can recreate the issue and resolve it again etc.

**Comment by Clax9** [comment]
@5P4C3M4N You're a genius!!! Thank you man :D

**Comment by 5P4C3M4N** [comment]
Glad to hear it worked for you too!

**Comment by rymmywife** [comment]
![image](https://cloud.githubusercontent.com/assets/20826953/17576006/979ab5c6-5f3c-11e6-9dca-eabd9dc74294.png)
IS this an IP ban?

**Comment by curtiola** [comment]
Looks that way yeah
